src/components/contact/Contact.jsx

```diff
@@ -1,33 +1,43 @@
-import React from "react";
+import React, { useMemo } from "react";
+
+const Contact = ({ name, number, email, city, status }) => {
+  const randomNumberGenerator = () => {
+    return Math.floor(Math.random() * 100000 + 1);
+  };
```

Comment:
create a folder in src called utils then put this util there in the file name called generateRandomNumber.js